docs/developer-guide/local-config.md

```diff
@@ -216,49 +216,6 @@ CSW service
 
 <br>
 
-### Measure Tool configuration
```

Comment:
it is no longer needed because was introduced the cfg.defaultOptions some time ago in the plugin configuration.
In general we should not use initialState.defaultState to configure plugins imho.